test/yarp_test_helper.rb

```diff
@@ -58,18 +58,19 @@ def assert_equal_nodes(expected, actual, compare_location: true, parent: nil)
           "start_offset > end_offset for #{actual.inspect}, parent is #{parent.pretty_inspect}"
         }
         if compare_location
-          assert_equal(
-            expected.start_offset,
-            actual.start_offset,
-            -> { "Start locations were different. Parent: #{parent.pretty_inspect}" }
-          )
-
-          assert_equal(
-            expected.end_offset,
-            actual.end_offset,
-            -> { "End locations were different. Parent: #{parent.pretty_inspect}" }
-          )
+          if !(parent.is_a?(YARP::StringNode) && (RbConfig::CONFIG['host_os'] =~ /mswin|mingw32|windows/))
```

Comment:
This looks good but would you mind moving `RbConfig::CONFIG['host_os'] =~ /mswin|mingw32|windows/)` into a constant? This function gets called a ton in the test suite.